iamproperty/iamproperty.github.io
# Update docs for breakpoints and grids

## Summary of Changes:
1. Documentation pages

## Rationale
Document for the grid system and breakpoints work.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /foundations/grid and /foundations/breakpoints

## Ticket(s)
DS-35
DS-34

## Checklist

## Notes
-
